commons/src/main/java/org/dhis2/commons/resources/D2ErrorUtils.kt

```diff
@@ -142,6 +142,7 @@ class D2ErrorUtils(private val context: Context) {
                 context.getString(R.string.error_invalid_characters)
             D2ErrorCode.PROGRAM_ACCESS_CLOSED ->
                 TODO("Implement for ANDROAPP-657")
+            D2ErrorCode.SERVER_CONNECTION_ERROR -> ""
```

Comment:
this error needs to be handled in https://dhis2.atlassian.net/browse/ANDROAPP-5396. maybe you can set a TODO here to avoid forget